playbooks/roles/edxapp/templates/edx/bin/edxapp-update-assets.j2

```diff
@@ -3,21 +3,29 @@
 export WEBPACK_CONFIG_PATH=webpack.prod.config.js
 export STATIC_ROOT_LMS=/edx/var/edxapp/staticfiles
 export STATIC_ROOT_CMS=/edx/var/edxapp/staticfiles/studio
+# This is necessary to properly build webworkers, which depends on this value being a dictionary.
+export JS_ENV_EXTRA_CONFIG={}
 
+
+# We need to make a call to a management command before running webpack because proctoring runs a webpack
+# webworker that only gets built if the proctoring djangoapp writes out a `workers.json`:
+# https://github.com/openedx/edx-proctoring/blob/73c7f55e2be91324fa07fec6e6ac0a667fdd8412/edx_proctoring/apps.py#L4
 {% if edxapp_staticfiles_storage_overrides %}
 {% for override in edxapp_staticfiles_storage_overrides %}
 export STATICFILES_STORAGE={{ override | quote }}
 sudo -E -H -u {{ edxapp_user }} \
     env "PATH=$PATH" \
-    npm run webpack \
+    {{ edxapp_venv_bin }} python manage.py lms --settings=$EDX_PLATFORM_SETTINGS print_setting STATIC_ROOT WEBPACK_CONFIG_PATH \
```

Comment:
This call (or the other one) caused a very unhelpful error during the Ansible run: `
env: ‘/edx/app/edxapp/venvs/edxapp/bin’: Permission denied`

Reply:
ohhh, I see the problem...
```suggestion
    {{ edxapp_venv_bin }}/python manage.py lms --settings=$EDX_PLATFORM_SETTINGS print_setting STATIC_ROOT WEBPACK_CONFIG_PATH \
```